package.json

```diff
@@ -63,6 +63,7 @@
   },
   "devDependencies": {
     "@edx/stylelint-config-edx": "1.1.0",
+    "babel-core": "^6.26.0",
```

Comment:
📌

😉

Reply:
Wait, why `babel-core` and not [`babel-plugin-transform-object-rest-spread`](https://babeljs.io/docs/plugins/transform-object-rest-spread/)?

Reply:
#17774 should swap this